src/components/item-display-panel/item-display-panel.component.ts

```diff
@@ -1,18 +1,24 @@
-import { Component, Directive, Input, SimpleChange, Output, EventEmitter } from '@angular/core';
+import { Component, Directive, Input, SimpleChange, Output, EventEmitter, Renderer2, ElementRef, Host } from '@angular/core';
 
 @Component({
     selector: 'ux-item-display-panel',
     templateUrl: './item-display-panel.component.html',
     host: {
         '(document:click)': 'clickOff($event)',
-        '(document:keyup.escape)': 'visible = false'
+        '(document:keyup.escape)': 'visible = false',
+        '[class.inlineHost]' : 'inline', 
+        '[class.visibleHost]' : 'visible' 
     }
 })
 export class ItemDisplayPanelComponent { 
     @Input() top: number;
     @Input() shadow: boolean;
     @Input() title: string;
     @Input() animate: boolean;
+    @Input() inline: boolean;
+    @Input() preventClose: boolean;
+    @Input() hideCloseButton: boolean;
```

Comment:
Our previous components tend to use `visible` for things like this, so perhaps change this to `closeVisible` instead